src/main/java/uk/gov/hmcts/cmc/claimstore/repositories/ClaimRepository.java

```diff
@@ -127,6 +127,17 @@ Integer linkLetterHolder(
     );
 
     @SqlUpdate(
+        "UPDATE claim SET sealed_claim_document_management_self_uri = :sealedClaimDocumentManagementSelfUri,"
+            + " sealed_claim_document_management_binary_uri = :sealedClaimDocumentManagementBinaryUri"
+            + " WHERE id = :claimId"
+    )
+    Integer linkDocumentManagement(
```

Comment:
can it return void? I assume you aren't looking at the result?

Reply:
IMHO I will preferred to know whether any row was successfully updated or not?